docs/submission/submitting-clinical-data.md

```diff
@@ -63,3 +63,48 @@ Uploaded data must be checked for accuracy against the data in the current submi
 1. If your data does not require approval by DCC, it will be submitted automatically.
 
 Once submitted, you will see the clinical data reflected on your [Program Dashboard](/docs/submission/submitted-data).
+
+## Submitting data in Surgery file
+
+#### How to indicate a donor who had Mastectomy surgery:
+
+- Submit 'treatment_type' = 'Surgery' and assign a unique submitter_treatment_id ('Tr-1' in this example):
+
+Treatment File
+![Treatment_Example1](/assets/submission/treatment_ex1.png)
+
+- Submit 'surgery_type' = 'Mastectomy' for the same submitter_treatment_id submitted in the Treatment file ('Tr-1), and indicate relevant details about surgery:
+
+Surgery File
+![Surgery Example1](/assets/submission/surgery_ex1.png)
+
+#### How to indicate a donor who had two surgeries at two different times.
+
+- Submit 'treatment_type' = 'Surgery' and assign a unique submitter_treatment_id to each surgery treatment event ('Tr-1' and 'Tr-2' in this example). Indicate treatment_start_interval and treatment_duration for each surgery treatment event.
+
+Treatment File
+![Treatment_Example2](/assets/submission/treatment_ex2.png)
+
+- Submit each surgery as two separate rows in the Surgery file (one for each submitter_treatment_id submitted in the Treatment file). Indicate relevant details about surgery.
+
+Surgery File
+![Surgery Example2](/assets/submission/surgery_ex2.png)
+
+#### How to indicate a specimen that was resected during a donor's mastectomy surgical procedure.
+
+- Indicate the unique identifier of the submitter_specimen_id ('Sp-32' in this example) that was resected during the mastectomy surgery. The submitter_specimen_id must exist in the Specimen file. You do not need to submit 'surgery_site' or 'surgery_location' since this information is collected in the Specimen table for that submitter_specimen_id.
```

Comment:
What happens if the submitter accidentally submits `surgery_site` and/or `surgery_location` in this case?  Will they be notified or is there an error?  If so do we need to let them know here?

Reply:
Yes, the data submitter will receive an error message like this:
![error](https://user-images.githubusercontent.com/4992044/173110282-75dc93b7-a7d3-4955-93ca-30a02e6c917d.png)

Reply:
Great, glad there is an error.  No issue, resolving.